src/factories/person/person.factory.js

```diff
@@ -88,3 +90,13 @@ export const retrievePersonOrgs = (personId) => {
   })
   return orgs
 }
+
+export const updatePerson = (personId, updatesToPerson) => {
+  if (!people[personId]) {
+    throw Boom.notFound(`Person with ID ${personId} not found`)
+  }
+
+  const person = people[personId] || createPerson(personId)
```

Comment:
`retrievePerson` will handle most of this for you...
```suggestion
  const person = retrievePerson(personId)
```